doc/en/user/source/installation/upgrade.rst

```diff
@@ -30,6 +30,14 @@ The general GeoServer upgrade process is as follows:
 Notes on upgrading specific versions
 ------------------------------------
 
+MapML Multi-Layer As Multi-Extent Configuration (GeoServer 2.28 and newer)
```

Comment:
Actually just "2.27 and newer", we are not even sure 2.28 will be released (we might have 3.0 in high gear by Septeber 2025, which is when 2.28 ought to be released ).

Reply:
Fixed